pkg/client/multi.go

```diff
@@ -372,11 +382,17 @@ func (m *MultiClient) ImageTag(ctx context.Context, image, tag string) error {
 }
 
 func (m *MultiClient) ImageDetails(ctx context.Context, imageName string, opts *ImageDetailsOptions) (result *ImageDetails, err error) {
-	c, err := m.Factory.ForProject(ctx, m.Factory.DefaultProject())
+	// Image may exist on any project within MultiClient
+	clients, err := m.Factory.List(ctx)
 	if err != nil {
 		return nil, err
 	}
-	return c.ImageDetails(ctx, imageName, opts)
+	for _, client := range clients {
+		if details, err := client.ImageDetails(ctx, imageName, opts); err == nil {
```

Comment:
This is causing the panic I shared in slack. In my case, that ImageDetails call returns an error, but we arent returning the error, we just fall out of the loop eventually and when we get to line 395, the `err` returned there is not the same err defined on this line

Reply:
fixed, ty!